.github/workflows/notify-docs-team.yml

```diff
@@ -0,0 +1,44 @@
+name: Notify Docs team
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  check:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    outputs:
+      files: ${{ steps.changes.outputs.files }}
+    steps:
+      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
+      - uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36
+        id: changes
+        with:
+          filters: |
+            files:
+              - '.changelog/**'
+              - 'website/**'
+        
+  slack-notification:
+    needs: check
+    if: ${{ needs.check.outputs.files == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: slackapi/slack-github-action@6c661ce58804a1a20f6dc5fbee7f0381b469e001
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL_TEST }}  # WILL CHANGE BEFORE MERGING TO: SLACK_WEBHOOK_URL_DOCS
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "hey @cloud-docs-on-call, this is :apix: bot, can you please review Terraform <${{ github.event.pull_request.html_url }}|PR ${{ github.event.pull_request.number }}>? thanks a lot!"
```

Comment:
in other slack notifications we set the tag to the oncall as a secret to not show it in the public repo ([link](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2052#discussion_r1535655077)). Would probably do the same here

Reply:
ok, will change